src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -318,6 +328,20 @@ protected virtual async Task VerifyCustomCommitProviderWorkerAsync(string codeBe
             }
         }
 
+        private bool TryParseXElement(string input, out XElement output)
+        {
+            try
+            {
+                output = XElement.Parse(input);
+                return true;
+            }
+            catch(XmlException)
+            {
+                output = default;
```

Comment:
💡 Use `null` instead of `default` when the type is known to be a reference type.

```suggestion
                output = null;
```